include/envoy/stream_info/stream_info.h

```diff
@@ -146,6 +146,18 @@ struct ResponseCodeDetailValues {
   // This will generally be accompanied by details about the transcoder failure.
   const std::string GrpcTranscodeFailed = "grpc_json_transcode_failure";
 
+  // The ext_authz filter denied the downstream request.
```

Comment:
@alyssawilk drive by comment: should these strings live with the extension? It seems like we shouldn't put extension reason strings in a core include file? WDYT?